packages/lesswrong/lib/collections/tagRels/collection.ts

```diff
@@ -5,6 +5,12 @@ import { makeVoteable } from '../../make_voteable';
 import { userCanUseTags } from '../../betas';
 
 const schema = {
+  createdAt: {
+    optional: true,
+    type: Date,
+    canRead: ['admins'],
```

Comment:
I think they should just be readable by guests, all the other createdAt fields are readably by guests